File: medium/two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        ans=[]
        l=0
        r=len(numbers)-1
        while l<r:
            if numbers[l]+numbers[r]==target:
                ans.append(l+1)
                ans.append(r+1)
                break
            elif numbers[r]+numbers[l]>target:
                r-=1
            else:
                l+=1
        return ans

File: medium/two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        vector<int> v;
        int l=0, r=size(numbers)-1;
        while (l<r) {
            if (numbers[r]+numbers[l]==target) {
                v.push_back(l+1);
                v.push_back(r+1);
                break;
            } else if (numbers[r]+numbers[l]>target) {
                r--;
            } else {
                l++;
            }
        }
        return v;
    }
};
